eppoclient/client.go

```diff
@@ -82,25 +95,30 @@ func (ec *EppoClient) GetJSONAssignment(flagKey string, subjectKey string, subje
 
 func (ec *EppoClient) getAssignment(flagKey string, subjectKey string, subjectAttributes Attributes, variationType variationType) (interface{}, error) {
 	if subjectKey == "" {
+		ec.applicationLogger.Error("no subject key provided")
 		panic("no subject key provided")
 	}
 
 	if flagKey == "" {
+		ec.applicationLogger.Error("no flag key provided")
 		panic("no flag key provided")
 	}
 
 	flag, err := ec.configRequestor.GetConfiguration(flagKey)
 	if err != nil {
+		ec.applicationLogger.Info("failed to get flag configuration: %v", err)
```

Comment:
`Info` level here but I can also see `Debug` being more appropriate. Nothing "wrong" happen here, this is normal behavior.